src/Sylius/Bundle/AdminBundle/Resources/views/Email/layout.html.twig

```diff
@@ -1 +1,19 @@
-{% extends '@SyliusCore/Email/layout.html.twig' %}
```

Comment:
Is there any reason we don't want to use this base template anymore?

Reply:
It has references to `SyliusShopBundle`. Examples:
`{% set logo = sylius_channel_url(asset('assets/shop/img/logo.png'), channel) %}`
```
{% if sylius_bundle_loaded_checker('SyliusShopBundle') %}
                        {% set url = sylius_channel_url(path('sylius_shop_homepage', {'_locale': localeCode}), channel) %}
                        <a href="{{ url|raw }}">
                            <img src="{{ logo }}" alt="Sylius" style="width: 170px">
                        </a>
                    {% else %}
                        <img src="{{ logo }}" alt="Sylius" style="width: 170px">
                    {% endif %}
```

I saw no point in passing channel (as in `sylius/sylius` there's no channel related to admin) so I've decided to make the `layout.html.twig` in `AdminBundle` independent.

Reply:
👍 aggreed. However, I would still propose this change in the separated PR (it does not influence only the newly created `adminPasswordReset.html.twig` but every Admin-related email), so we should make this fix explicit and noticeable. For now, we can just not extend any template in `adminPasswordReset.html.twig` and go with propose layout in it 🚀